lib/servers/iXonServer/AndorVideo.py

```diff
@@ -1,4 +1,5 @@
-from config.andor_config import andor_config as config
+import andor_config as config
```

Comment:
This is not the standard config format. If we are staying with the old format we should need no changes